game/scripts/vscripts/GameMode.ts

```diff
@@ -152,15 +153,6 @@ export class GameMode {
             Timers.CreateTimer(0.2, () => GameRules.FinishCustomGameSetup());
         }
 
-        if (state === GameState.HERO_SELECTION) {
-            Timers.CreateTimer(0.2, () => {
```

Comment:
auto hero selection removed?

Reply:
As discussed, it is used with `this.Game.SetCustomGameForceHero("npc_dota_hero_dragon_knight");`
All that did was assign the player with the hero, which we no longer need since it is assigned to the gamemode entity now and is correctly fetched using `getPlayerHero()`.